### PR TITLE
[CI] pr-feedback: Add start-date, exempt-authors and exempt-labels

### PR DESCRIPTION
**Pull request by marcelklehr**

### CODE REVIEW

**Review by DaphneMuller — APPROVED** [review]

File: .github/workflows/pr-feedback.yml [hunk 1]
```diff
@@ -8,15 +8,23 @@ jobs:
...
-            We hope that the reviewing process is going smooth and is helpful for you. We want to ensure your pull request is reviewed to your satisfaction. If you have a moment, we would appreciate your feedback on your experience with our community management team. 
+            We hope that the reviewing process is going smooth and is helpful for you. We want to ensure your pull request is reviewed to your satisfaction. If you have a moment, our community management team would very much appreciate your feedback on your experience with this PR reviewing process.
 
             Your feedback is valuable to us as we continuously strive to improve our community developer experience. Please take a moment to complete our short survey by clicking on the following link: https://cloud.nextcloud.com/apps/forms/s/i9Ago4EQRZ7TWxjfmeEpPkf6
 
             Thank you for contributing to Nextcloud and we hope to hear from you soon!
           days-before-feedback: 14
+          start-date: "2023-05-01"
+          exempt-authors: "${{ steps.scrape.outputs.users }}"
+          exempt-labels: "dependencies"
```
Comment (provokateurin) [inline .github/workflows/pr-feedback.yml:30]: Backports?

Comment (marcelklehr) [reply .github/workflows/pr-feedback.yml:30]: Ah, good point!

**Review by provokateurin — APPROVED** [review]

File: .github/workflows/pr-feedback.yml [hunk 2]
```diff
@@ -8,15 +8,23 @@ jobs:
...
-            We hope that the reviewing process is going smooth and is helpful for you. We want to ensure your pull request is reviewed to your satisfaction. If you have a moment, we would appreciate your feedback on your experience with our community management team. 
+            We hope that the reviewing process is going smooth and is helpful for you. We want to ensure your pull request is reviewed to your satisfaction. If you have a moment, our community management team would very much appreciate your feedback on your experience with this PR reviewing process.
 
             Your feedback is valuable to us as we continuously strive to improve our community developer experience. Please take a moment to complete our short survey by clicking on the following link: https://cloud.nextcloud.com/apps/forms/s/i9Ago4EQRZ7TWxjfmeEpPkf6
 
             Thank you for contributing to Nextcloud and we hope to hear from you soon!
           days-before-feedback: 14
+          start-date: "2023-05-01"
+          exempt-authors: "${{ steps.scrape.outputs.users }},backportbot-nextcloud"
+          exempt-labels: "dependencies"
```
Comment (provokateurin) [inline .github/workflows/pr-feedback.yml:30]: Maybe it's better to exempt dependabot than the dependencies label? There might be manual PRs that are tagged with that label

Comment (marcelklehr) [reply .github/workflows/pr-feedback.yml:30]: True

**Pull request merged**